tests/libres_tests/res/fm/test_ecl_run_new.py

```diff
@@ -118,6 +118,29 @@ def test_run(self):
         errors = ecl_run.parseErrors()
         self.assertEqual(0, len(errors))
 
+    @tmpdir()
+    @pytest.mark.equinor_test
+    def test_run_new_log_file(self):
```

Comment:
Is is necessary to have two test functions with the same name and even the same (almost) code?

Reply:
Yes, one tests the eclrun codepath, the other tests the old style formatting.